clients/cmd/docker-driver/config.go

```diff
@@ -66,18 +66,16 @@ const (
 	defaultHostLabelName  = model.LabelName("host")
 )
 
-var (
-	defaultClientConfig = client.Config{
-		BatchWait: client.BatchWait,
-		BatchSize: client.BatchSize,
-		BackoffConfig: backoff.Config{
-			MinBackoff: client.MinBackoff,
-			MaxBackoff: client.MaxBackoff,
-			MaxRetries: client.MaxRetries,
-		},
-		Timeout: client.Timeout,
-	}
-)
+var defaultClientConfig = client.Config{
+	BatchWait: client.BatchWait,
+	BatchSize: client.BatchSize,
+	BackoffConfig: backoff.Config{
+		MinBackoff: client.MinBackoff,
+		MaxBackoff: client.MaxBackoff,
+		MaxRetries: client.MaxRetries,
```

Comment:
I mean here

Reply:
Interesting. The default is `10` so the driver should not deadlock it can just take almost an hour until the client gives up. For production use this should be fine.

Reply:
Yeah like what I call a deadlock nobody wants to wait more than 5s :)